src/renderer/components/screens/AccountListSidebar/AccountListSidebar.tsx

```diff
@@ -0,0 +1,119 @@
+import { T } from '@deltachat/jsonrpc-client'
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import { BackendRemote } from '../../../backend-com'
+import { AccountItem } from './AccountItem'
+
+import styles from './styles.module.scss'
+import { AccountHoverInfo } from './AccountHoverInfo'
+import { debounce } from 'debounce'
+import { runtime } from '../../../runtime'
+
+export function AccountListSidebar({
```

Comment:
I'd be up for moving all new components just into the `components` folder (no `screens` subfolder, etc.), at least for everything new. It's hard to understand the semantics of `screens`, is `AccountListSidebar` really a screen?

Reply:
it is at least on the same upper level in code, but sure we can change the folder structure, your comment makes sense.